manager/integration/tests/common.py

```diff
@@ -1466,7 +1466,18 @@ def wait_for_replica_failed(client, volname, replica_name):
 
 @pytest.fixture
 def volume_name(request):
-    return generate_volume_name()
+    volume_name = generate_volume_name()
```

Comment:
Is it safe to change this? It seems lots of assumption is made here and this function has been widely used in the test cases.